feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.hedvig.android.odyssey.search
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.animation.Crossfade
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.auth.android.AuthenticatedObserver
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.ui.appbar.CenterAlignedTopAppBar
+import com.hedvig.android.odyssey.OdysseyClaimsFlowActivity
+import org.koin.androidx.viewmodel.ext.android.getViewModel
+
+class SearchActivity : ComponentActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    lifecycle.addObserver(AuthenticatedObserver())
+
+    setContent {
+      val viewModel: SearchViewModel = getViewModel()
+      val viewState by viewModel.viewState.collectAsState()
+
+      val commonClaimId = viewState.selectedClaim?.id
+      if (commonClaimId != null) {
+        startClaimsFlow(viewModel, commonClaimId)
+      }
```

Comment:
This'd need to be:
```suggestion
      LaunchedEffect(commonClaimId) {
        if (commonClaimId != null) {
          startClaimsFlow(viewModel, commonClaimId)
        }
      }
```
Since otherwise `startClaimsFlow()` is called in composition, which may be called more than once without you wanting it to. 

Kinda covered in [this section](https://developer.android.com/jetpack/compose/mental-model#simple-example) in the docs which asks for composable functions to be idempotent, meaning if called again with the same inputs, it shouldn't change anything, in which case here it will, by calling startClaimsFlow() again without the commonClaimId changing.